src/main/42-apigw.tf

```diff
@@ -5,7 +5,7 @@ locals {
 resource "aws_api_gateway_rest_api" "apigw" {
   name        = "${var.app_name}-apigw-${var.env}"
   description = "${var.app_name} API Gateway"
-  body        = file(local.openapi_spec)
+  body        = templatefile(local.openapi_spec, { external_authorizer_arn = var.external_authorizer_uri, cognito_authorizer_arn = var.cognito_authorizer_uri })
```

Comment:
Why pass these values as variables? You can get (or build) them from TF resources

Reply:
The authorizer object is defined and created in the openapi specification

Reply:
Those variables contain arns that refer to lambda functions that we create in TF, can't we build the arns using the functions' attributes?

Reply:
Done